test/spec/escapesequences.js

```diff
@@ -1119,7 +1121,19 @@
         }
       ]
     },
-    "a = [[\\]]": {
+    "a = '\\.'": {
+      "result": "[1:7] invalid escape sequence near '\\.'",
+      "options": {
+        "scope": true,
+        "locations": true,
+        "ranges": true,
+        luaVersion: 5.2
```

Comment:
This quoting style inconsistency suggests that you typed this in manually. Didn't you?

This file is supposed to be automatically generated from `test/scaffolding/escapesequences`. You should modify that, and then run `make scaffold-tests` in the root directory. Options can be specified in a line of the form `//{...}` (look into `scripts/scaffold-test` for details).